src/core/internal/util/math.d

```diff
@@ -0,0 +1,53 @@
+// Written in the D programming language
+
+/**
+ * Internal math utilities.
+ *
+ * Copyright: Luís Ferreira 2021
```

Comment:
I'm not really sure if this should be my or D Programming Language Foundation copyrights. Can someone please clarify it to me?

Reply:
Should be DLF. Old code only has it for historical reasons.

Reply:
Force pushed :)